KachnaOnline.App/Controllers/EventsController.cs

```diff
@@ -32,12 +33,12 @@ public EventsController(EventsFacade eventsFacade)
         /// Returns a list of events being held at the given date and time.
         /// </summary>
         /// <param name="at">The date and time.</param>
-        /// <returns>An enumerable of <see cref="EventDto"/> corresponding to the <paramref name="at"/> date and time.</returns>
+        /// <returns>A list of <see cref="EventDto"/> corresponding to the <paramref name="at"/> date and time.</returns>
         /// <response code="200">The list of events.</response>
         [AllowAnonymous]
         [HttpGet("at/{at}")]
         [ProducesResponseType(typeof(IEnumerable<EventDto>), StatusCodes.Status200OK)]
-        public async Task<IActionResult> GetEvents(DateTime at)
+        public async Task<ActionResult<IEnumerable<EventDto>>> GetEvents(DateTime at)
```

Comment:
As I've said before, the sole (_more or less_) purpose of `ActionResult<T>` is to simplify returning the primary return type of the controller. When it's an `IEnumerable<T>`, one has to either encapsulate it with that weird `new ActionResult<IEnumerable<T>>(t)` or use `Ok(t)` – but then there's (_almost_) no point in specifying the return type as `ActionResult<T>`.

_Yes, it is a bit messy;_ however, using `IActionResult` in this specific case is preferred. 

I didn't change the signature back when I was merging the first version of this file because I had a lot of spare time.

Reply:
Thank you for the explanation. Will change. Though, I must admit, I did not notice you changing these, so I was wondering where did these come from and I have changed them back again to what I thought should be there. My bad.